### ISSUE TITLE
Why kv_cache_size = min(local_attn_size + slice_last_frames, num_training_frames) in training?

### ISSUE THREAD

**Issue by Candy-Crusher** [NVlabs/LongLive#23]
<img width="1040" height="376" alt="Image" src="https://github.com/user-attachments/assets/379203ef-9f05-49df-8038-15d44c1e6037" />
In config, local_attn_size=12, slice_last_frames=21, num_training_frames=21. Therefore the kv_cache_size=21 in training, while it is 12 in inference. Does it mean that there is no kv_cache rolling in training? I don't quite understand why slice_last_frames is used here.

And in self_attn, self_max_attention_size is set as local_attn_size=12, does this lead to a mismatch?

**Comment by AndysonYs** [comment]
Hi. This design is for training loss backpropagation. In the 2nd training stage, local_attn_size=12, slice_last_frames=18, num_training_frames=240. You can check Section3.3-"Consistency between Training and Inference" in our paper for details
